### ISSUE TITLE
Icon request: fa-passkey

### ISSUE THREAD

**Issue by newbold** [FortAwesome/Font-Awesome#19405]
As the Passkey standard is being adopted by Apple, Google, and Microsoft, it would be nice to have an icon for it. There isn’t yet an official “brand” icon for it, but it appears that a consistent icon pattern of "user icon + vertical key" is emerging. The icon could be created relatively easily by combining fa-user and fa-key (with the key rotated slightly in a vertical orientation).

Reference image:

![passkeys-128x128_2x](https://user-images.githubusercontent.com/986721/198402675-3c537c95-30e9-454c-9e7a-669f2fae2b3a.png)

More examples:

 - https://www.passkeys.io
 - https://developer.apple.com/passkeys/
 - https://developers.google.com/identity/passkeys

**Comment by newbold** [comment]
Also, apologies for not using the correct template — I followed the link from [https://github.com/FortAwesome/Font-Awesome/blob/6.x/CONTRIBUTING.md](this page) and wound up on the issue submission form without the intended template. Sorry!

**Comment by tagliala** [comment]
No problem, Thanks for spotting the wrong link!

**Comment by markusramsak** [comment]
Are there any plans to continue drafting a passkey font awesome icon? On more and more websites I offer a Passkey Login option where I would like to use a dedicated icon instead of fingerprint.

**Comment by Dygear** [comment]
Is there a bounty that can be paid to get this into the next version? I agree with @markusramsak - It seems awkward to me to use the fingerprint icon for what really should be the passkey logo.

**Comment by dnorthup-ums** [comment]
(1) It is not appropriate to use the fingerprint icon for passkey authentication. They are two meaningfully different things, as passkeys do not depend strictly upon biometrics (fingerprints, etc) for authentication.
(2) The passkey logo legal is food for thought: https://fidoalliance.org/overview/legal/ & https://fidoalliance.org/get-the-passkey-icon/ (With a use & style guide here: https://fidoalliance.org/wp-content/uploads/2023/12/FIDO-Passkey_Icon_Usage_Guidelines-August2022.pdf )
(3) Presumably the icons in https://github.com/passkeydeveloper/passkeys.dev/tree/main/static/images have the blessing the FIDO Alliance, but it may be worth checking with them on that.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.

**Comment by Dygear** [comment]
For resubmitting this for discussion, it says that the person should understand what makes something a brand or not. I don't think this is a brand request, it's more of a common mark for a technology use. Are there guidelines for what is and is not a brand, because this one seems a little murky based on the tags attached to this issue. My feeling is that it's not a brand.

**Comment by sensibleworld** [comment]
@Dygear I tend to agree. Looks like this may have been incorrectly labeled originally. We do have this icon already: https://fontawesome.com/icons/user-key?f=classic&s=solid

(Though it looks like it could use a "passkey" search term)

**Comment by newbold** [comment]
No need to submit for discussion if the icon already exists (obviously), but how do we propose updating the search metadata?

**Comment by sensibleworld** [comment]
I've already got it updated in our system. Should be reflected when we next do a release.
